content/docs/user-guide/data-management/index.md

```diff
@@ -0,0 +1,105 @@
+# Data Management with DVC
+
+Storing and transferring datasets and ML models can vary depending on project
+needs, available infrastructure, etc. DVC helps you avoid logistics like object
+permissions on cloud storage, sync tools and schedules, back up snapshots, etc.
+and focus on machine learning.
+
+You work with data normally in a local <abbr>workspace</abbr>. DVC tracks,
+restores, and synchronize everything with a few, straightforward operations that
+do not change regardless of the underlying file systems, transfer protocols,
+etc.
+
+![]() _Separating data from code (codification)_
+
+<details>
+
+## Click to learn more about data _codification_
+
+To achieve this, DVC replaces large files and directories with small [metafiles]
+that describe the assets. Data files are moved to a separate <abbr>cache</abbr>
+but kept virtually (linked) in the workspace. This separates your data from code
+(including metafiles).
+
+<admon type="tip">
+
+This also allows you to [version] project files with Git, a battle-tested [SCM]
+tool.
+
+</admon>
+
+[version]: /doc/user-guide/data-management/data-versioning
+[scm]: https://www.atlassian.com/git/tutorials/source-code-management
+
+</details>
+
+Your experience can stay consistent because DVC works [indirectly], by checking
+the [metafiles] and [configuration] of your <abbr>project</abbr> to find out
+where and how to handle files. This is transparent to you as user, but it's
+important to understand the mechanics in general.
+
+[metafiles]: /doc/user-guide/project-structure
+[indirectly]: https://en.wikipedia.org/wiki/Indirection
+[configuration]: /doc/command-reference/config
+
+## Workflow and benefits
```

Comment:
Let's for now iterate only on this part (and even remove everything else from PR). 

Here - how can we explain it better, draw diagrams?

Things like `Stored objects are organized by DVC and you don't need to touch them directly; DVC exposes a few commands to manage them;` are not very informative (it's more or less clear that it's managed by DVC?)

Reply:
> `... and you don't need to touch them directly`
...not very informative

This comes from what I understood from our last chat (see https://github.com/iterative/dvc.org/pull/4042#discussion_r1008451492). Did you mean instead to emphasize that the cache structure is used in all storage locations managed by DVC? (Not considering cloud versioning for now) So more like "you're not supposed to touch them" ? I tried to phrase it in a positive way.

Anyway let me work on this part, currently it's very [drafty](https://github.com/iterative/dvc.org/pull/4042#pullrequestreview-1161038043) ⏳

Reply:
OK, I rewrote it again. It's still long and it may need removing sections, diagrams, etc. still but in better shape than a draft now, I think. PTAL